torchvision/transforms/functional_tensor.py

```diff
@@ -19,10 +17,8 @@ def vflip(img_tensor):
 
 def hflip(img_tensor):
     """Horizontally flip the given the Image Tensor.
-
```

Comment:
Can you revert the whitespace changes all over the file? They are necessary for the documentation to render properly